cluster-autoscaler/cloudprovider/azure/azure_kubernetes_service_pool.go

```diff
@@ -315,7 +315,7 @@ func (agentPool *AKSAgentPool) DeleteNodes(nodes []*apiv1.Node) error {
 
 //IsAKSNode checks if the tag from the vm matches the agentPool name
 func (agentPool *AKSAgentPool) IsAKSNode(tags map[string]*string) bool {
-	poolName := tags["poolName"]
+	poolName := tags["aks-managed-poolname"]
```

Comment:
can we move this to a const at the start of the file? The tag should be `aks-managed-poolName`